recipes/cuda-nvcc/bld.bat

```diff
@@ -0,0 +1,17 @@
+if not exist %PREFIX% mkdir %PREFIX%
```

Comment:
```suggestion
if not exist %PREFIX% mkdir %PREFIX%
if not exist %LIBRARY_PREFIX% mkdir %LIBRARY_PREFIX%
```

Reply:
I'm unfamiliar with Windows, is this something we should be doing in all of our recipes? [We didn't do that in libcublas](https://github.com/conda-forge/libcublas-feedstock/blob/main/recipe/bld.bat), for instance. We're not specifically writing anything to the `%LIBRARY_PREFIX%` directory in this file.